packages/@okta/vuepress-site/docs/reference/api/uischema/index.md

```diff
@@ -0,0 +1,748 @@
+
+`okta.uischemas.manage`
+
+#### Request path parameters
+
+| Parameter | Type   | Description                                           |
+| --------- | ------ | ----------------------------------------------------- |
+| `id`      | String | The unique ID of the UI Schema |
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+N/A
+
+#### Response body
+
+N/A
+
+#### Usage example
+
+The following request deletes a UI Schema with an `id` value of `uis4a97f4pmZsdxKu0g7`:
+
+##### Request
+
+```bash
+curl -v -X DELETE \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/meta/uischemas/uis4a97f4pmZsdxKu0g7"
+```
+
+##### Response
+
+Returns an empty HTTP 204 status code response
+
+#### Error Responses
+
+If an invalid UI Schema ID is included in the request, the response returns a 404 NOT FOUND with error code E0000008.
+
+## UI Schema API objects
+
+### UI Schema Request object
+
+#### UI Schema Request properties
+
+| Property   | Type | Description |
+| ---------- | ------------------------------------- | ----------------------------------------------------------- |
+| `id`       | String                                | The unique ID of the ui schema |
+| `uiSchema` | [UI Schema Object](#ui-schema-object) |  |
+
+#### UI Schema Request example
+
+### UI Schema Response object
+
+#### UI Schema Response object properties
+
+| Property      | Type                                                                    | Description                                                                                                       |
+| ------------- | ----------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `id`          | String                                                                  | The unique ID of the UI schema |
+| `uiSchema`    | [UI Schema Object](#ui-schema-object)  |  |
+| `lastUpdated` | String (ISO-8601) | Timestamp when the UI Schema was last modified |
+| `_links`      | Object ([JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06)) | Discoverable resources related to the UISchema |
+
+#### UI Schema Response object example
+
+```json
+{
+  "id": "uis4a97f4pmZsdxKu0g7",
+  "uiSchema": {},
+  "created": "2022-07-28T13:02:25.000Z",
+  "lastUpdated": "2022-07-28T14:31:42.000Z",
+  "_links": {
+    "self": {
+      "href": "https://dev-jake-2.trexcloud.com/api/v1/meta/uischemas/uis4a97f4pmZsdxKu0g7",
+      "hints": {
+        "allow": ["GET", "PUT", "DELETE"]
+      }
+    }
+  }
+}
+```
+
+### UI Schema Object
+
+#### UI Schema properties
+
+| Property      | Type   | Description | Supported Values |
+| ------------- | ------ | ----------- | ---------------- |
+| `buttonLabel` | String | Specifies the button label for the `submit` button at the bottom of the enrollment form. Defaults to `Submit`. |
+| `elements`    | Array  |  |
```

Comment:
@jakebacher2-okta There are a few properties in the "UI Schema object" section that are missing descriptions. They still had the template placeholder content which I deleted.

Reply:
Not sure what the ask is here. If it is for `elements` missing it's description, we should point it to the `### UI Schema Element object` section below I believe.

Reply:
Ah. Thank you!